src/NewWorldMinimap/Program.cs

```diff
@@ -8,13 +8,18 @@ namespace NewWorldMinimap
     /// </summary>
     public static class Program
     {
+        //https://stackoverflow.com/questions/32438736/taking-a-screenshot-using-graphics-copyfromscreen-with-150-scaling/37800963
```

Comment:
I don't think it's very relevant to link this, since this is just a native windows function.